CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Change Log
 ===
 
+v3.2.0-dev.2
+---
+* [bugfix] formatters are now exported correctly (#863)
```

Comment:
be a little more explicit and say "... to work with TS 1.8"

by --amending, you'll also retrigger the build and it should pass now that I've released grunt-tslint 3.0.1